### PR TITLE
Tweaks Adminordrazine, Cleans up Reagents, Fixes body temperature adjustments, and makes a few tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Tweaks adminordrazine a bit

Chem Tweaks
- Adminordrazine
 - chem purging removed
 - heals 5 internal organ damage
 - attempts to mend fractures
 - uses proper proc for adjusting sleeping
 - won't cure beneficial viruses

- fixes cold and hot drinks: they should now properly heat you up or cool you down (this also goes for medical reagents)
-  beer no longer reduces jitteriness
- toxins special and antifreeze heat you up
- banana juice and banana honk now heals monkeys
- Sprinkles now works for every member of security (sec pod pilot, IAA, magistrate, and brig phys are no longer left out)
- Adds in vitamin reagent; restores blood better than nutriment and can heal brute and burn but doesn't restore as much nutrition

Other Changes
- Removed lexorin from the mining pens
- Stimpack is now meth+coffee instead of meth+epinephrine
- Combat hypospray now has epinephrine+omnizine+teporone
- Cheese and synth meat reactions scale with volume (more volume = more meat/cheese wheels)

Code
- cleans up some arguments containing `var` in them.
- removes trailing returns
- ensures `..()` is always called last
- proper `AdjustSleeping` or `SetSleeping` proc usage
- Axes all `data` in favor of `current_cycle` (for tracking a cycle, that is)
- Removes most relative pathing from reagent code (minus food section...another day)
- Reagent containers (again minus food) use `reagents_list` now as opposed to using `reagents.add_reagent` in `New`
- Removed unused borg syringes and bottles
- Removes as lot of `as mob` and `as mob|obj` crap
- Ensures things are pathed `/datum/reagent` instead of `datum/reagent`
- Removes a bunch of commented out code
- Removes unused `alien` argument
- Pushes the `if(!M) M = holder.my_atom ` into metabolize (yay, even less copy-pasta)
- Removes unneeded living check in `metabolize`
- Removes Xenoarch reagents
 - This included the coolant recipe as well---none of these are used anyway. Good riddance for axing Cael stuff.
- Refactors nutriment, protein, and plant-matter; protein and plant matter are now children of nutriment; yay, less copy-pasta again
- Removes `check_gofast` and `check_goreallyfast`; the proper status flags are just cleared when the reagent is deleted

:cl: Fox McCloud
tweak: nanites will no longer purge chems, but will now heal internal organ damage, attempt to mend fractures, and will no longer cure beneficial viruses
fix: cold and hot drinks should now properly heat you up or cool you down (this also goes for medical reagents)
tweak: beer no longer reduces jitteriness
tweak: toxins special and antifreeze heat you up
tweak: banana juice and banana honk now heals monkeys
fix: Fixes atrazine not killing nymphs
tweak: mining autoinjectors no longer poison you
tweak: combat hypospray now has omni+epinephrine+teporone
tweak: stimpack is now meth+coffee instead of meth+epinephrine
tweak: Cheese and synth meat amount creation scales with volume of the reaction
tweak: sprinkles now heals all members of sec (brig phys, sec pod pilot, magistrate, and IAA are no longer left out)
/:cl:

### CODE REVIEW

**Comment by LittleBigKid2000** [comment]
\>admin only chem
\>balance

**Comment by PJB3005** [comment]
\> Removing chem purging from the 1 reagent that's literally supposed to be god mode.

???

**Comment by Fox-McCloud** [comment]
well, I didn't originally intend for this to be what it was...buttt

**Comment by Earthdivine** [comment]
>Bananas heal Monkeys
>Does nothing to Humans.

**Comment by KasparoVy** [comment]
y'all gone bananas

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -128,11 +121,7 @@
 	volume = 40
 	spray_maxrange = 4
 	amount_per_transfer_from_this = 5
-
-
-/obj/item/weapon/reagent_containers/spray/pepper/New()
-	..()
-	reagents.add_reagent("condensedcapsaicin", 40)
+	list_reagents = list("condensedcapsaicin" = 50)
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/spray.dm]: needs to be 40

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 2]
```diff
@@ -209,86 +177,64 @@
...
+	list_reagents = list("diethylamine" = 50)
 
 /obj/item/weapon/reagent_containers/glass/bottle/reagent/acid
 	name = "Acid Bottle"
 	desc = "A reagent bottle. Contains sulfuric acid."
 	icon_state = "rbottle6"
-	New()
-		..()
-		reagents.add_reagent("sacid", 50)
+	list_reagents = list("diethylamine" = 50)
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: should be "sacid"

File: code/modules/reagents/reagent_containers/drugs.dm [hunk 3]
```diff
@@ -94,39 +95,45 @@
...
 
 /obj/item/weapon/reagent_containers/drugs/baggie/meth
-	New()
-		..()
-		reagents.add_reagent("methamphetamine", 10)
\ No newline at end of file
+
+/obj/item/weapon/reagent_containers/drugs/New()
+	..()
+	reagents.add_reagent("methamphetamine", 10)
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/drugs.dm]: should be `list_reagents = list("methamphetamine" = 10)`

File: code/modules/reagents/reagent_containers/drugs.dm [hunk 4]
```diff
@@ -12,80 +12,81 @@
 
 	var/label_text = ""
 
-	New()
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/drugs.dm]: this whole thing is shit. why wasn't this made a subtype of glass with different fillings/updat_icons. ughhhhh

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 5]
```diff
@@ -286,60 +284,6 @@
...
-	var/syringestab_amount_transferred = rand(0, (reagents.total_volume - 5)) //nerfed by popular demand
-	src.reagents.trans_to(target, syringestab_amount_transferred)
-	src.desc += " It is broken."
-	src.mode = SYRINGE_BROKEN
-	src.add_blood(target)
-	src.add_fingerprint(usr)
-	src.update_icon()
-
-
 /obj/item/weapon/reagent_containers/ld50_syringe
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/syringes.dm]: le :snowflake:

File: code/modules/reagents/oldchem/reagents/reagents_food.dm [hunk 6]
```diff
@@ -24,54 +23,44 @@
 				else
 					if(!(H.species.flags & NO_BLOOD))
 						H.vessel.add_reagent("blood", 0.4)
-		if(istype(M,/mob/living/simple_animal))		//Any nutrients can heal simple animals
```
Comment (Fox-McCloud) [inline code/modules/reagents/oldchem/reagents/reagents_food.dm]: I dunno who added this or why--simple animals have never processed reagents.

**Pull request merged**